handler/area_types.go

```diff
@@ -0,0 +1,52 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/ONSdigital/dp-cantabular-dimension-api/contract"
+	"github.com/ONSdigital/dp-cantabular-dimension-api/model"
+	"github.com/pkg/errors"
+)
+
+const (
+	ParamCantabularBlob = "cantabular_blob"
+)
+
+// AreaTypes handles requests to /area-types
+type AreaTypes struct {
+	respond responder
+	ctblr   cantabularClient
+}
+
+// NewAreaTypes returns a new AreaTypes handler
+func NewAreaTypes(r responder, c cantabularClient) *AreaTypes {
+	return &AreaTypes{
+		respond: r,
+		ctblr:   c,
+	}
+}
+
+// Get is the handler for GET /area-types
+func (at *AreaTypes) Get(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	blob := r.URL.Query().Get(ParamCantabularBlob)
```

Comment:
We can handle decoding query strings in a cleaner way by using `https://github.com/gorilla/schema`, which will essentially allow us to create RequestObjects as we do for POST bodies. Something like:

```
import "https://github.com/gorilla/schema"

type GetAreaTypesRequest struct{
  PopulationType string `schema:"population_type"`// As per Andrew's comment below we should not be using cantabular_blob
}
var req GetAreaTypesRequest
if err := schema.NewDecoder().Decode(&req, r.URL.Query()); err != nil{
...
`